.github/workflows/ci.yml

```diff
@@ -180,16 +180,21 @@ jobs:
     needs: build
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
```

Comment:
Smoke tests are mysteriously failing on `windows-latest`. We'll need to investigate separately